lib/cursor.js

```diff
@@ -216,21 +216,21 @@ if (SUPPORTS.ASYNC_ITERATOR) {
 
 // Map core cursor _next method so we can apply mapping
 Cursor.prototype._next = function() {
-  if (this._initImplicitSession) {
-    this._initImplicitSession();
-  }
   return CoreCursor.prototype.next.apply(this, arguments);
 };
 
 for (let name in CoreCursor.prototype) {
   Cursor.prototype[name] = CoreCursor.prototype[name];
 }
 
-Cursor.prototype._initImplicitSession = function() {
+Cursor.prototype._initializeCursor = function(callback) {
```

Comment:
Does this not need to be copied to Agg cursor and Command cursor?

Reply:
`_initializeCursor` is already copied for the [CommandCursor](https://github.com/mongodb/node-mongodb-native/blob/next/lib/command_cursor.js#L155), and the `AggregationCursor` just outright subclasses the core `Cursor`, so they both have it:
```
> const AggCursor = require('./lib/aggregation_cursor');
> const testCursor = new AggCursor({ s: { options: {} } }, 'foo.test', { limit: 10 }, {}, {})
undefined
> testCursor._initializeCursor
[Function]
```